### PR TITLE
[Rule Tuning] Update Rule Content Changes from Security Docs Team

### PR DESCRIPTION
**Pull request by terrancedejesus**
## Issues
Resolves https://github.com/elastic/detection-rules/issues/1944

## Summary

- linux-restricted-shell-breakout-via-the-vi-command
- potential-remote-credential-access-via-registry
- signed-proxy-execution-via-ms-workfolders
- suspicious-remote-registry-access-via-sebackupprivilege
- user-account-exposed-to-kerberoasting
- windows-registry-file-creation-in-smb-share
- linux-restricted-shell-breakout-via-crash-shell-evasion
- linux-restricted-shell-breakout-via-env-shell-evasion
- linux-restricted-shell-breakout-via-flock-shell-evasion
- linux-restricted-shell-breakout-via-the-expect-command
- linux-restricted-shell-breakout-via-the-find-command
- linux-restricted-shell-breakout-via-the-gcc-command
- linux-restricted-shell-breakout-via-the-mysql-command
- linux-restricted-shell-breakout-via-the-ssh-command

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Review by imays11 — COMMENTED** [review]
Most of these look good. Dates are all reset and you've changed the content of the description field for each. The only one I'm confused by is the description in the 'execution_expect_binary'. It seems like the content has been mixed up with information from "find" and "vi". But this rule is specific to the expect binary correct? If so, the description doesn't seem to match.

File: rules/linux/execution_expect_binary.toml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 author = ["Elastic"]
 description = """
-Identifies Linux binary expect abuse to break out from restricted environments by spawning an interactive system shell.
-The expect utility allows us to automate control of interactive applications such as telnet,ftp,ssh and others and the
-activity of spawning shell is not a standard use of this binary for a user or system administrator and could potentially
-indicate malicious actor attempting to improve the capabilities or stability of their access.
+Identifies Linux binary find abuse to break out from restricted environments by spawning an interactive system shell.
+The vi/vim editor is the standard text editor in Linux distributions, and the activity of spawning a shell is not a
+standard use of this binary by a user or system administrator. This could potentially indicate a malicious actor
+attempting to improve the capabilities or stability of their access.
```
Comment (terrancedejesus) [inline rules/linux/execution_expect_binary.toml:12]:
```suggestion
Identifies Linux binary expect command abuse to break out from restricted environments by spawning an interactive system shell. The expect utility allows us to automate control of interactive applications such as Telnet, FTP, SSH and others. The activity of spawning shell is not a standard use of this binary for a user or system administrator and could potentially indicate malicious actor attempting to improve the capabilities or stability of their access.
```

File: rules/linux/execution_flock_binary.toml [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 author = ["Elastic"]
 description = """
-Identifies Linux binary flock abuse to break out from restricted environments by spawning an interactive system
-shell.The flock utility allows us to manage advisory file locks in shell scripts or on the command line and the activity
-of spawing a shell is not a standard use of this binary by a user or system administrator. It indicates a potentially
-malicious actor attempting to improve the capabilities or stability of their access.
+Identifies Linux binary env abuse to break out from restricted environments by spawning an interactive system shell. The
+env utility is a shell command for Unix-like operating systems and is used to print a list of environment variables. The
+activity of spawning a shell is not a standard use of this binary for a user or system administrator. It indicates a
+potentially malicious actor attempting to improve the capabilities or stability of their access.
```
Comment (terrancedejesus) [inline rules/linux/execution_flock_binary.toml:12]:
```suggestion
Identifies Linux binary flock abuse to break out from restricted environments by spawning an interactive system shell. The flock utility allows users to manage advisory file locks in shell scripts or on the command line. The activity of spawning a shell is not a standard use of this binary by a user or system administrator. It indicates a potentially malicious actor attempting to improve the capabilities or stability of their access.
```

**Comment by terrancedejesus** [comment]
> Most of these look good. Dates are all reset and you've changed the content of the description field for each. The only one I'm confused by is the description in the 'execution_expect_binary'. It seems like the content has been mixed up with information from "find" and "vi". But this rule is specific to the expect binary correct? If so, the description doesn't seem to match.

Good catch! Updated shell evasion flock binary and shell evasion expect binary rules

File: rules/linux/execution_expect_binary.toml [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 [rule]
 author = ["Elastic"]
 description = """
-Identifies Linux binary expect abuse to break out from restricted environments by spawning an interactive system shell.
-The expect utility allows us to automate control of interactive applications such as telnet,ftp,ssh and others and the
-activity of spawning shell is not a standard use of this binary for a user or system administrator and could potentially
-indicate malicious actor attempting to improve the capabilities or stability of their access.
+Identifies Linux binary expect command abuse to break out from restricted environments by spawning an interactive system
+shell. The expect utility allows us to automate control of interactive applications such as Telnet, FTP, SSH and others.
+The activity of spawning shell is not a standard use of this binary for a user or system administrator and could
```
Comment (imays11) [inline rules/linux/execution_expect_binary.toml:11]: "The activity of spawning a shell is not a standard use of this binary by a user or system administrator. It indicates a potentially malicious actor attempting to improve the capabilities or stability of their access."

-This is the language I see being used in the other rules. I'm assuming you meant to change it here as well.

**Review by imays11 — CHANGES_REQUESTED** [review]

File: rules/linux/execution_expect_binary.toml (hunk 3 above)
Comment (terrancedejesus) [reply rules/linux/execution_expect_binary.toml:11]: Not sure I follow? That language is included here from what I see.

**Review by w0rk3r — APPROVED** [review]

File: rules/windows/credential_access_spn_attribute_modified.toml [hunk 4]
```diff
@@ -77,11 +77,11 @@ DS Access >
 Audit Directory Service Changes (Success,Failure)
 ```
 
-The above policy does not cover User objects, so we need to set up an AuditRule using https://github.com/OTRF/Set-AuditRule.
+The above policy does not cover User objects, so set up an AuditRule using https://github.com/OTRF/Set-AuditRule.
```
Comment (Mikaayenson) [inline rules/windows/credential_access_spn_attribute_modified.toml:80]:
```suggestion
The above policy does not cover User objects, so configure an AuditRule using https://github.com/OTRF/Set-AuditRule.
```

**Review by Mikaayenson — APPROVED** [review]
LGTM. 

There was on small recommendation for consistency with the other updates.

File: rules/linux/execution_crash_binary.toml [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
 [rule]
 author = ["Elastic"]
 description = """
-Identifies Linux binary crash abuse to break out from restricted environments by spawning an interactive system
-shell.The crash utility helps to analyze Linux crash dump data or a live system and the activity of spawing a shell is
-not a standard use of this binary by a user or system administrator. It indicates a potentially malicious actor
-attempting to improve the capabilities or stability of their access.
+Identifies Linux binary crash abuse to break out from restricted environments by spawning an interactive system shell.
+The crash utility helps analyze Linux crash dump data or a live system and the activity of spawning a shell is not a
+standard use of this binary by a user or system administrator. It indicates a potentially malicious actor attempting to
```
[the remark this answers is not in the document]
Comment (terrancedejesus) [reply rules/linux/execution_crash_binary.toml:11]: Thanks for recommendation and review! These are only meant to mimic what the security docs are for 8.2 based on the review from the Security Docs team.

File: rules/windows/credential_access_spn_attribute_modified.toml (hunk 4 above)
Comment (terrancedejesus) [reply rules/windows/credential_access_spn_attribute_modified.toml:80]: Thanks for recommendation. These are only meant to mimic what the security docs are for 8.2 based on the review from the Security Docs team.

**Review by imays11 — APPROVED** [review]

**Pull request merged**